### PR TITLE
Implement SEP6 /transaction and /transactions endpoints

### LINKED ISSUE #16: [SEP 6] Implement the /transactions endpoint

### LINKED ISSUE #17: [SEP 6] Implement the /transaction endpoint
#

### PR DESCRIPTION
**Pull request by accordeiro**
This PR:
- Refs #14 
- Closes #16 
- Closes #17 

It aims to:
- [x] Create initial Transaction database / ORM models
- [x] Implement the `/transactions` endpoint and its filtering parameters
- [x] Add tests for the format of the `/transactions` endpoint
- [x] Add tests for the filtering parameters of the `/transactions` endpoint
- [x] Implement the `/transaction` endpoint and its filtering parameters
- [x] Add tests for the format of the `/transaction` endpoint
- [x] Add tests for the filtering parameters of the `/transaction` endpoint

**Note for reviewers:** the business logic is consolidated in `src/transaction/models.py` and `src/transaction/views.py`. I've also added extensive tests to illustrate how to use the API and ensure we can gracefully handle edge cases.

### CODE REVIEW

File: src/info/migrations/0002_auto_20190726_2032.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Generated by Django 2.2.3 on 2019-07-26 20:32
```
Comment (accordeiro) [inline src/info/migrations/0002_auto_20190726_2032.py]: **Note:** This migration was generated automatically, and doesn't include any database-level changes. Safe to ignore.

File: src/transaction/views.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    """
+
+    try:
+        limit = _validate_limit(request.GET.get("limit"))
+    except ValueError:
+        return Response({"error": "invalid limit"}, status=status.HTTP_400_BAD_REQUEST)
+
+    translation_dict = {
+        "asset_code": "asset__name",
+        "account": "stellar_account",
```
Comment (tomquisel) [inline src/transaction/views.py]: It should probably error out if no account is defined.

File: src/transaction/models.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        choices=STATUS, default=STATUS.pending_external, max_length=30
+    )
+    status_eta = models.IntegerField(null=True, blank=True)
+    stellar_transaction_id = models.TextField(null=True, blank=True)
+    external_transaction_id = models.TextField(null=True, blank=True)
+    amount_in = models.FloatField(null=True, blank=True)
+    amount_out = models.FloatField(null=True, blank=True)
+    amount_fee = models.FloatField(null=True, blank=True)
+    started_at = models.DateTimeField(auto_now_add=True)
+    completed_at = models.DateTimeField(null=True)
```
[the remark this answers is not in the document]
Comment (accordeiro) [reply src/transaction/models.py]: Cool, thanks for the heads up! I've just added all new v3 fields :)

File: src/transaction/views.py [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+        "no_older_than": "started_at__lte",
```
[the remark this answers is not in the document]
Comment (accordeiro) [reply src/transaction/views.py]: Good catch. Just fixed that!

File: src/transaction/views.py (hunk 2 above)
Comment (accordeiro) [reply src/transaction/views.py]: Yep, you're right. Actually, SEP6 lists both `asset_code` and `account` as required, so I've updated the implementation / tests accordingly.
